cmake/tribits/ReleaseNotes.txt

```diff
@@ -2,35 +2,41 @@
 Release Notes for TriBITS
 ----------------------------------------
 
+2021/03/12:
+
+(*) MAJOR: Upgrade minimum required CMake version from 3.10 to 3.17.  Existing
+    TriBITG projects that have already upgraded to require CMake 3.17+ should not
```

Comment:
tribits is misspelled

Reply:
> tribits is misspelled

Thanks @rppawlo!  This is fixed in TriBITS commit https://github.com/TriBITSPub/TriBITS/commit/3731f72f5654e0d88970038667241c08fea68845.  This will get pulled into Trilinos in the next TriBITS snapshot.